src/kontrol/options.py

```diff
@@ -434,6 +440,24 @@ def get_argument_type() -> dict[str, Callable]:
             }
         )
 
+    def apply_optimizations(self) -> None:
+        """Applies a series of performance optimizations based on the value of the
+        `optimize_performance` attribute.
+
+        If `optimize_performance` is not `None`, this method will adjust several
+        internal parameters to enhance performance. The integer value of `optimize_performance`
+        is used to set the level of parallelism for frontier exploration and maintenance rate.
+        """
+        if self.optimize_performance is not None:
+            self.assume_defined = True
+            self.log_succ_rewrites = False
+            self.max_frontier_parallel = self.optimize_performance
+            self.maintenance_rate = 2 * self.optimize_performance
+            self.smt_timeout = 120000
```

Comment:
Is it not 32000 (as in @PetarMax's message) intentionally? :)

Reply:
@PetarMax  suggested the change to `120000` in a comment above.